dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/statelisteners/ElasticListener.java

```diff
@@ -482,6 +484,27 @@ private static Set<String> getVerifiedPlatforms(Set<? extends Version> workflowV
         return platforms;
     }
 
+    private static List<String> getDistinctDescriptorTypeVersions(Entry entry, Set<? extends Version> workflowVersions) {
+        String language;
+        if (entry instanceof Tool && ((Tool)entry).getDescriptorType().size() == 1) {
```

Comment:
There's an interesting Java 16 syntax pointed out in Sonar cloud. 
May be useful to try out and may be worth running an intellj refactoring on the whole codebase if supported (can be separate PR)